contracts/BondFactory.sol

```diff
@@ -137,7 +156,25 @@ contract BondFactory is AccessControl {
             collateralToken,
             collateralRatio,
             convertibleRatio,
-            maxSupply
+            maxSupply,
+            collateralDeposited
         );
     }
+
+    function _deposit(
+        address owner,
+        address clone,
+        address collateralToken,
+        uint256 collateralToDeposit
+    ) internal returns (uint256) {
```

Comment:
nit: may want to name the return var

Reply:
Great idea! Fixed [baf260c](https://github.com/porter-finance/v1-core/pull/175/commits/baf260c53755137a1b3cde4c76b114fa5a3c41e6)